Misc/NEWS.d/next/Core and Builtins/2020-08-28-20-54-04.bpo-1635741.7ijlcI.rst

```diff
@@ -0,0 +1 @@
+Port the :mod:`zlibmodule` extension module to multi-phase initialization (:pep:`489`).

```

Comment:
```suggestion
Port the :mod:`zlib` extension module to multi-phase initialization (:pep:`489`).

```

Reply:
@vstinner I assumed it was convention for the filename to be the module name but now I see that based on the PyInit_* signatures it is not always the case.  I'll keep this in mind